kong/clustering/rpc/json_rpc_v2.lua

```diff
@@ -32,7 +32,7 @@ function _M.new_error(id, code, msg)
     id = id,
     error = {
       code = code,
-      message = msg,
+      message = tostring(msg),
```

Comment:
And we should panic once the `err` cannot be converted to a meaningful `string`, such as a `table` without metamethod `__tostring`.

Reply:
done.